src/Microsoft.TemplateEngine.Edge/Settings/SettingsLoader.cs

```diff
@@ -211,7 +243,12 @@ public bool TryGetMountPoint(string mountPointUri, out IMountPoint mountPoint)
             {
                 throw new ObjectDisposedException(nameof(SettingsLoader));
             }
-            return _mountPointManager.TryDemandMountPoint(mountPointUri, out mountPoint);
+            return MountPointManager.TryDemandMountPoint(mountPointUri, out mountPoint);
+        }
+
+        public void ResetSettings()
+        {
+            _paths.Delete(_paths.BaseDir);
```

Comment:
Can we rename this `BaseDir` to be same as property in `IPathInfo.cs`, from this name I don't know if this is `~/.templateengine/` or `~/.templateengine/dotnetcli/v6.0.100/`...

Reply:
Also... If we want to do this rename... Can it be different PR?

Reply:
will do in next PR